src/component/footer/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
```

Comment:
![image](https://user-images.githubusercontent.com/70959734/186361692-5c609786-ba72-4859-9205-aa2e6ceaf768.png)
why did you create a folder for the footer component?

Reply:
i think it is not necessary to create folder here

Reply:
use folder when you have separate component belong parent component